src/styles/index.scss

```diff
@@ -4,0 +1,4 @@
+@import '/src/styles/utils/variables';
+@import '/src/styles/body';
+@import '/src/styles/product-card';
+@import '/src/styles/stars';
```

Comment:
Ensure that the file `/src/styles/stars.scss` exists and is correctly referenced. This file should contain the styles for the star rating component.